docs/runtimes/application.ipynb

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
<blockquote>The application runtime deploys a user container image (for example, a web application), a port, and a command to run the HTTP server. </blockquote><p>This sentence is not accurate, as it doesn't "deploy a port and a command", it deploys a container image that is exposed on a specific port.</p><p> </p><blockquote>or use a pre-built image</blockquote><p>As Alon said, this is not supported.</p><p> </p><blockquote>you can disable it to allow free access without authentication. </blockquote><p>The authentication can be disabled, not the API Gateway. Maybe something like:</p><p>"... is in front of the application and can provide different authentication methods, or none"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>